golden-ratio.el

```diff
@@ -94,15 +95,13 @@ will not cause the window to be resized to the golden ratio."
                    (loop for fun in golden-ratio-inhibit-functions
                          thereis (funcall fun))))
     (let ((dims (golden-ratio--dimensions))
-          (golden-p (if golden-ratio-mode 1 -1)))
+          (golden-ratio-mode nil))
```

Comment:
Roman Gonzalez <notifications@github.com> writes:

> In golden-ratio.el:
>
>> @@ -94,15 +95,13 @@ will not cause the window to be resized to the golden ratio."
>>                     (loop for fun in golden-ratio-inhibit-functions
>>                           thereis (funcall fun))))
>>      (let ((dims (golden-ratio--dimensions))
>> -          (golden-p (if golden-ratio-mode 1 -1)))
>> +          (golden-ratio-mode nil))
>
> What is the difference between using nil and -1, and any particular
> reason this was moved to the let statement and not keep it where it
> was before?

IIUC they have problems with the hydra package and doing so instead
of disabling the whole mode avoid recursion.
I can't give you more details because I don't use the hydra package.

-- 
Thierry
Get my Gnupg key:
gpg --keyserver pgp.mit.edu --recv-keys 59F29997